src/seismometer/plot/mpl/likert.py

```diff
@@ -0,0 +1,222 @@
+                    ha="center",
+                    va="center",
+                    fontsize=10,
+                )
+            elif width >= 5:
+                ax.text(
+                    bar.get_x() + width / 2,
+                    bar.get_y() + bar.get_height() / 2,
+                    f"{percentage:.0f}%",
+                    ha="center",
+                    va="center",
+                    fontsize=10,
+                )
+    ax.set_yticks(range(len(df.index)))
+    ax.set_yticklabels(_wrap_labels(df.index), fontsize=12)
+    ax.set_ylim(-0.5, len(df.index) - 0.5)
+    # Add legend
+    ax.legend(loc="upper right", bbox_to_anchor=(1.2, 1))
+    max_left = abs(df_percentages[first_half_columns].sum(axis=1) - middle_adjustment).max()
+    max_right = abs(df_percentages[second_half_columns].sum(axis=1) - middle_adjustment).max()
+    ax.set_xlim(-max_left - border, max_right + border)
+    # Add labels and title
+    ax.set_xlabel("Percentages of Responses", fontsize=12)
+    ax.xaxis.set_major_locator(MaxNLocator(nbins=10, integer=True))
+    ax.xaxis.set_major_formatter(FuncFormatter(lambda x, _: f"{int(x)}%"))
+    ax.tick_params(axis="x", labelsize=12)
+    ax.set_title(title)
+    return fig
+
+
+def _plot_counts(df: pd.DataFrame, ax_count: matplotlib.axes.Axes, border: int = 5):
+    """
+    Plots the counts of each row in the dataframe as horizontal bars.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        DataFrame containing the counts of each category.
+    ax_count : matplotlib.axes.Axes
+        The axes on which to plot the counts.
+    border : int, optional
+        Border space around the plot, by default 5.
+    """
+    total_counts = df.sum(axis=1)
+    bars = ax_count.barh(df.index, total_counts, height=0.7, color=area_colors[0])
+    max_count = max(total_counts)
+    for bar in bars:
+        width = bar.get_width()
+        if width >= 0.1 * max_count:
+            ax_count.text(
+                bar.get_x() + width / 2,
+                bar.get_y() + bar.get_height() / 2,
+                f"{_format_count(width)}",
+                ha="center",
+                va="center",
+                fontsize=10,
+                color="black",
+            )
+        else:
+            ax_count.text(
+                bar.get_x() + width + 0.01 * max_count,
+                bar.get_y() + bar.get_height() / 2,
+                f"{_format_count(width)}",
+                ha="left",
+                va="center",
+                fontsize=10,
+                color="black",
+            )
+    ax_count.set_yticks(range(len(df.index)))
+    ax_count.set_yticklabels(_wrap_labels(df.index), fontsize=12)
+    ax_count.set_xlabel("Counts", fontsize=12)
+    ax_count.set_title("Counts of Each Row")
+    ax_count.set_xlim(0, total_counts.max() + border)
+    ax_count.xaxis.set_major_locator(MaxNLocator(nbins=5))
+    ax_count.xaxis.set_major_formatter(FuncFormatter(lambda x, _: f"{int(x)}%"))
```

Comment:
Results in  counts with % at the end

![image](https://github.com/user-attachments/assets/d44e2dde-c84d-4237-a2f5-4aa6f5b907c9)

```suggestion
    ax_count.xaxis.set_major_formatter(FuncFormatter(_format_count))
```

Reply:
FINC.